Code/EnginePlugins/PhysXPlugin/WorldModule/Implementation/PhysXWorldModule.cpp

```diff
@@ -785,9 +790,11 @@ void ezPhysXWorldModule::FetchResults(const ezWorldModule::UpdateContext& contex
 
     if (cvar_PhysicsDebugDrawEnable)
     {
-      m_pPxScene->setVisualizationParameter(PxVisualizationParameter::eCOLLISION_SHAPES, 1);
-      m_pPxScene->setVisualizationParameter(PxVisualizationParameter::eCONTACT_POINT, 1);
-      m_pPxScene->setVisualizationParameter(PxVisualizationParameter::eCOLLISION_DYNAMIC, 1);
+      //m_pPxScene->setVisualizationParameter(PxVisualizationParameter::eCOLLISION_SHAPES, 1);

```

Comment:
Should all of these be disabled?

Reply:
Yes, this is a bit WIP. Sometimes I need these, sometimes I need others. But currently I don't want to add 20 CVars for all the possibilities.